### DISCUSSION TITLE
TimeTable Generator

### DISCUSSION THREAD

**Discussion by just-injoey** [scaleracademy/open-source-projects#155] [Ideas]
We all know there are many time table generators currently present in the market, but still there are many flaws in all the exisiting application, as in most of these apps, we just spend our time creating the time table, but end of the day it doesn't improve our time management.
It should also have a well designed and easy to use To-Do list. 
Both the To-Do list and TimeTable generator shpuld work smoothly in sync.

Tasks to be done:
- Create a time table generator
- Create a ToDo list which should work in sync with time tabl generator
-  We can either create an app or a website

**Comment by nayanapardhekar** [comment]
Superb idea. Actually should work on it.
